File: OddNumberPyramid/oddNumberPyramid.js
var x= prompt("Enter a number of rows: ");
var output="";

//specfies nb of columns
function count(x){
    for(let i=1; i<=x; i++){
        if(i%2!==0){
            output+=i+" "
        }
    }
    console.log(output);
    output="";
}

//specifies number of rows
function drawPyramid(x){
    for(let i=1; i<=2*x-1; i=i+2){
        count(i);
    }
    console.log("");
}

drawPyramid(x);

File: OddNumberPyramid/oddNumberPyramid.py
x= int(input("Enter a number of rows: "))

def count(x):
    output=""
    for i in range(1,x+1):
        if i%2!=0:
            output+=str(i)+" "

    print(output)

def drawPyramid(x):
    for i in range(1,2*x,2):
        count(i)
    print("")

drawPyramid(x)